apps/nameguard.io/src/components/organisms/Header.tsx

```diff
@@ -34,33 +35,32 @@ export const Header = () => {
         <div className="flex flex-row items-center justify-between md:gap-5 h-[40px]">
           <div className="hidden items-center justify-center lg:flex gap-1">
             <a
-              className="px-[10px] py-[9px] not-italic text-black font-medium text-sm leading-5 rounded-md sm:hover:bg-gray-100"
               href="https://api.nameguard.io/docs"
               target="_blank"
               rel="noopener noreferrer"
             >
-              Docs
+              <Button variant="ghost" size="small">
+                Docs
+              </Button>
             </a>
+
             <a
-              className="rounded-md sm:hover:bg-gray-100"
               href="https://github.com/namehash/nameguard"
               target="_blank"
               rel="noopener noreferrer"
             >
-              <div className="flex flex-row justify-between items-center gap-2 py-[7px] px-[10px]">
+              <Button variant="ghost" size="small">
                 <GithubIcon className="hidden md:block text-[#0F172A] fill-current" />
-                <p className="not-italic text-black font-medium text-sm leading-5">
-                  Github
-                </p>
-              </div>
+                Github
+              </Button>
             </a>
-            <Link
-              className="px-[10px] py-[9px] not-italic text-black font-medium text-sm leading-5 rounded-md gt_mobile:hover:bg-gray-100"
-              href="/contact"
-              rel="noopener noreferrer"
-            >
-              Contact
-            </Link>
+
+            <a href="/contact" rel="noopener noreferrer">
```

Comment:
Why `noopener noreferrer` here? This is an internal link. Generally we should only do that on external links.